src/server/cluster/cluster_family.cc

```diff
@@ -35,15 +35,7 @@ using namespace std;
 using namespace facade;
 using CI = CommandId;
 
-void BuildClusterSlotNetworkInfo(ConnectionContext* cntx, std::string_view host, uint32_t port,
-                                 std::string_view id) {
-  constexpr unsigned int kNetworkInfoSize = 3;
-
-  (*cntx)->StartArray(kNetworkInfoSize);
-  (*cntx)->SendBulkString(host);
-  (*cntx)->SendLong(port);
-  (*cntx)->SendBulkString(id);
-}
+constexpr string_view kClusterNotConfigured = "-Cluster is not yet configured";
```

Comment:
why did you add the '-' char at the beginning?

Reply:
I should not have, sorry!